modules/objdetect/include/opencv2/objdetect/aruco_calib_pose.hpp

```diff
@@ -0,0 +1,286 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef __OPENCV_ARUCO_CALIB_POSE_HPP__
+#define __OPENCV_ARUCO_CALIB_POSE_HPP__
+#include <opencv2/objdetect/board.hpp>
+#include <opencv2/calib3d.hpp>
+
+namespace cv {
+namespace aruco {
+
+//! @addtogroup aruco
+//! @{
+
+/** @brief rvec/tvec define the right handed coordinate system of the marker.
+ * PatternPos defines center this system and axes direction.
+ * Axis X (red color) - first coordinate, axis Y (green color) - second coordinate,
+ * axis Z (blue color) - third coordinate.
+ * @sa estimatePoseSingleMarkers(), @ref tutorial_aruco_detection
+ */
+enum PatternPos {
```

Comment:
Let's stay calib and extra things in contrib for now and make the migration with small steps. Please migrate aruco detector with tests and  docs only with this PR. The same for samples, if they use detector only. Calib thing requires small redesign.